lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -285,12 +357,22 @@ def handle(self, *args, **options):
             student_dict['Gender'] = student_dict['Gender'].upper()
 
             csvwriter.writerow(student_dict)
-
             count += 1
-
+        print user_scores
         outfile.close()
         sys.stdout.write("Data written to {name}\n".format(name=outfile_name))
 
+
+    def get_scores(self, user, course_id):
+        """
```

Comment:
This is the function that we need to future proof. At least for current stanford/master, the new rebase target and a ginkgo version